demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -139,8 +151,9 @@ def __len__(self):
 
 
 def get_model(ie, args):
+    common_args = (ie, args.model, args.reverse_input_channels, args.mean_values, args.scale_values)
```

Comment:
IMO, the latter three values should be combined into a class (e.g. `ModelInputTransforms`) and passed around as an object of that class. It would be clearer than adding 3 positional arguments to every model constructor.

Reply:
Done.